### PR TITLE
WFLY-9436 Test suite misses any sort of smoke test for JSR-77 management bean availability

### PR DESCRIPTION
**Pull request by ChesterSVK**
Created simple JSR77 test for MBean functionality.
JIRA :
https://issues.jboss.org/browse/WFLY-9436

### CODE REVIEW

**Review by rhusar — CHANGES_REQUESTED** [review]
Always make sure to reference Jira ID in the SCM commit message.

**Comment by kabir** [comment]
This is ok to test

**Review by stuartwdouglas — CHANGES_REQUESTED** [review]
This test fails.

**Comment by ChesterSVK** [comment]
Fixed

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsr77/Jsr77TestCase.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package org.jboss.as.test.integration.jsr77;
```
[the remark this answers is not in the document]
Comment (ChesterSVK) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsr77/Jsr77TestCase.java]: Added

**Comment by bstansberry** [comment]
retest this please

**Pull request merged**